chirp/drivers/ft450d.py

```diff
@@ -1001,162 +816,198 @@ def get_settings(self):
+        tmp = "CAT RTS flow control"
+        rs = RadioSetting("cat_rts", tmp, rx)
         rs.set_apply_callback(self._invert_me, _settings, "cat_rts")
         tab.append(rs)
 
         options = ["0", "100ms", "1000ms", "3000ms"]
-        rs = RadioSetting("cat_tot", "CAT Timeout",
-                          RadioSettingValueList(options,
-                          options[_settings.cat_tot]))
+        optn = options[_settings.cat_tot]
+        rx = RadioSettingValueList(options, optn)
+        tmp = "CAT Timeout"
+        rs = RadioSetting("cat_tot", tmp, rx)
         tab.append(rs)
 
         options = ["4800", "9600", "19200", "38400", "Data"]
-        rs = RadioSetting("catrate", "CAT rate",
-                          RadioSettingValueList(options,
-                          options[_settings.catrate]))
-        tab.append(rs)
-
-        rs = RadioSetting("mem_grp", "Mem groups",
-                          RadioSettingValueBoolean(_settings.mem_grp))
+        optn = options[_settings.catrate]
+        rx = RadioSettingValueList(options, optn)
+        tmp = "CAT rate"
+        rs = RadioSetting("catrate", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("scn_res", "Resume scan (secs)",
-                          RadioSettingValueInteger(0, 10, _settings.scn_res))
+        rx = RadioSettingValueBoolean(_settings.mem_grp)
+        tmp = "Mem groups"
+        rs = RadioSetting("mem_grp", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("clk_sft", "CPU clock shift",
-                          RadioSettingValueBoolean(_settings.clk_sft))
+        val = _settings.scn_res
+        rx = RadioSettingValueInteger(0, 10, val)
+        tmp = "Resume scan (secs)"
+        rs = RadioSetting("scn_res", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("split", "TX/RX Frequency Split",
-                          RadioSettingValueBoolean(_settings.split))
+        rx = RadioSettingValueBoolean(_settings.clk_sft)
+        tmp = "CPU clock shift"
+        rs = RadioSetting("clk_sft", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("qspl_f", "Quick-Split freq offset (KHz)",
-                          RadioSettingValueInteger(-20, 20, _settings.qspl_f))
+        rx = RadioSettingValueBoolean(_settings.split)
+        tmp = "TX/RX Frequency Split"
+        rs = RadioSetting("split", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("emergen", "Alaska Emergency Mem 5167.5KHz",
-                          RadioSettingValueBoolean(_settings.emergen))
+        val = _settings.qspl_f
+        rx = RadioSettingValueInteger(-20, 20, val)
+        tmp = "Quick-Split freq offset (KHz)"
+        rs = RadioSetting("qspl_f", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("stby_beep", "PTT release 'Standby' beep",
-                          RadioSettingValueBoolean(_settings.stby_beep))
+        rx = RadioSettingValueBoolean(_settings.emergen)
+        tmp = "Alaska Emergency Mem 5167.5KHz"
```

Comment:
it's kHz with a lowercase k and there should be a space before the unit: 5167.5 kHz